libnmstate/nm/ipv4.py

```diff
@@ -96,6 +97,14 @@ def get_info(active_connection):
     if active_connection is None:
         return info
 
+    nm_ac_state = nmclient.NM.ActiveConnectionState
+    if active_connection.get_state() == nm_ac_state.ACTIVATING:
+        logging.debug(
+            'Interface {} is in activating state, showing saved config '
+            'in of running state '.format(active_connection.get_id())
+        )
+        return _get_config(active_connection)
```

Comment:
This feels wrong. We will report what is desired by NM, but it may or may not be actual.
Perhaps we need to limit this to very specific scenarios. For example, if we asked dynamic IP, then we can report the config of those specific parameters and nothing more.